src/main/java/uk/co/ramp/Properties.java

```diff
@@ -0,0 +1,45 @@
+package uk.co.ramp;
+
+import uk.co.ramp.io.DiseaseProperties;
+import uk.co.ramp.io.PopulationProperties;
+import uk.co.ramp.io.StandardProperties;
+import uk.co.ramp.io.readers.DiseasePropertiesReader;
+import uk.co.ramp.io.readers.PopulationPropertiesReader;
+import uk.co.ramp.io.readers.StandardPropertiesReader;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Properties {
+
+    private static StandardProperties standardProperties;
+    private static DiseaseProperties diseaseProperties;
+    private static PopulationProperties populationProperties;
+
+    private Properties() {
+
+    }
+
+    public static StandardProperties getStandardProperties() throws IOException {
+        if (standardProperties == null) {
```

Comment:
Why are you using lazy loading for these properties?
(I would expect you'd try loading them all at the beginning, and fail early if any are missing.)

Reply:
+1

Reply:
This class has been removed.